package.json

```diff
@@ -105,6 +105,7 @@
     "watchify": "^3.7.0"
   },
   "dependencies": {
+    "@flamingbear/cmrjs": "1.10.4-rc.8",
```

Comment:
`@flamingbear/cmrjs`? why not `@cumulus/cmrjs`

Reply:
@jennyhliu we need a new release of cumulus 1.10.4 before we can change this to @cumulus/cmrjs   This was a temporary npm package to ease testing.  (so that you didn't have to check out the branch on cumulus repo and link the npm packages).  The ticket says to move this to blocked after approval and update these to point to the real cumulus/cmrjs package.